### ISSUE TITLE
"VNC connection failed!" When trying to connect with androidVNC

### ISSUE THREAD

**Issue by number-g** [CypherpunkArmory/UserLAnd#422]
When I try to open a VNC session, I get:

"VNC connection failed!"
"Authentication failed!"

This is using an LG Nexus 5x and it happens whether I select Debian or Ubuntu.

SSH works fine.

Any Idea how to resolve this?

Many thanks,

Greg

**Comment by dsvilko** [comment]
Is the VNC password set correctly? Try changing it with the vncpasswd command. Are you sure that the server is running. For some reason yesterday my xfce4 session would no longer start and my tigervncserver would exit. I have fixed the problem by putting the xfce4-session command in my .vnc/Xvnc-session file. The default file in /etc/X11 didn't run correctly.

**Comment by corbinlc** [comment]
I agree with what @maxolasersquad said.  Right now we only officially support bVNC.  But if someone would like to take a look at the androidVNC intent code vs the bVNC intent code, maybe they can shed some light on what is different.

**Comment by corbinlc** [comment]
Closing as part of a repository cleanup. If this issue is still relevant, feel free to reopen it or file a fresh one.
